test/typescript/securityValidation.test.js

```
@@ -0,0 +1,179 @@
+
+describe("Security validation", () => {
+  const originalNodeEnv = process.env.NODE_ENV
+  const originalStrictValidation = process.env.SHAKAPACKER_STRICT_VALIDATION
+
+  afterEach(() => {
+    process.env.NODE_ENV = originalNodeEnv
+    process.env.SHAKAPACKER_STRICT_VALIDATION = originalStrictValidation
+    clearValidationCache()
+  })
+
+  describe("Path traversal security checks", () => {
```

ESLint

[error] `describe`s should begin with lowercase

(jest/prefer-lowercase-title)